examples/tutorials/Introduction to Langrangian Neural Networks.ipynb

```diff
@@ -0,0 +1,824 @@
+{
```

Comment:
<p>Rename the functions, make the names intuitive and simple but descriptive. Add docstrings to explain what the functions do, 1-2 lines will do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3905/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>